src/lib/Token.tsx

```diff
@@ -82,3 +104,70 @@ export function getTokenIcon(token: Token) {
       return <BonkIconCircle />;
   }
 }
+
+export function getTokenId(token: Token) {
+  switch (token) {
+    case Token.SOL:
+      return "solana";
+    case Token.mSOL:
+      return "msol";
+    case Token.stSOL:
+      return "lido-staked-sol";
+    case Token.USDC:
+      return "usd-coin";
+    case Token.USDT:
+      return "tether";
+    case Token.RAY:
+      return "raydium";
+    case Token.ORCA:
+      return "orca";
+    case Token.Bonk:
+      return "bonk";
+  }
+}
+
+export function tokenAddressToToken(address: string) {
```

Comment:
lets make a token database file with all necessary metadata fields per token. you can pull all of them using jupiter api